tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -550,6 +551,7 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_config_add_cert_chain_and_key_to_store(server_config, ecdsa_cert));
         /* Override auto-chosen defaults with only RSA cert default. ECDSA still loaded, but not default. */
         EXPECT_SUCCESS(s2n_config_set_cert_chain_and_key_defaults(server_config, &rsa_cert, 1));
+        EXPECT_SUCCESS(s2n_connection_set_config(conn, server_config));
```

Comment:
> This can be fixed by associating valid configs with the connections before using them in tests.

I think I'm missing something. Why is this test not a valid way to use a config? Why is `s2n_connection_set_config` needed here?

Reply:
Ohhh I think I see now. So the config was associated with a connection and then freed and the connection is still pointing to the freed config here.